src/router/guards/data.js

```diff
@@ -23,14 +23,13 @@ export default async function dataGuard() {
     // 1) Load current user data
     await loadUser();
 
-    // 2) Load organizations, spaces and projects of the current user
-    await Promise.all([
-      loadUserOrganizations(),
-      loadUserSpaces(),
-      loadUserProjects()
-    ]);
+    // 2) Load organizations of the current user
+    await loadUserOrganizations();
```

Comment:
no because we have added in space mapper a new property called isUserOrga.
The mapper is used when we call the loadUserSpaces function, and it needs a data that is returned by loadUserOrganizations